src/opcodes/tests/test_opcodes.cairo

```diff
@@ -1068,3 +1068,21 @@ fn test_op_1negate() {
     let expected_stack = array![ScriptNum::wrap(-1)];
     assert_eq!(dstack, expected_stack.span(), "Stack is not equal to expected");
 }
+
+#[test]
+fn test_op__0_not_equal() {
+    let program = "OP_1 OP_0NOTEQUAL";
+    let mut compiler = CompilerTraitImpl::new();
+    let bytecode = compiler.compile(program);
+    let mut engine = EngineTraitImpl::new(bytecode);
+
+    engine.step();
+    let res = engine.step();
+    assert!(res, "Execution of OP_ABS failed for negative number");
```

Comment:
Some of these comments need to be updated to describe this test, think this was an artifact from copying another test setup.